### ISSUE TITLE
Chain down views stop gap 

### ISSUE THREAD

**Issue by gamarin2** [EmerisHQ/demeris#418]
We won't have time to implement https://github.com/allinbits/demeris/issues/197 before launch. We are going to have https://github.com/allinbits/demeris/pull/385 which will help, but it doesn't solve an issue that Peng showed: duplicate balances. If a chain is down, balances stop getting updated, meaning if a user does a transfer using another UI than ours, their balance might appear duplicated. Properly dealing with this requires quite a bit of work. In the meantime, we have two options:

## Option 1

A simple solution is the following: implement a global banner when one chain is down (cf https://www.figma.com/file/04OF6ZbTo0gLzmFm3kLRkS/app.emeris.com?node-id=4925%3A73468)

<img width="1157" alt="Capture d’écran 2021-08-11 à 01 11 20" src="https://user-images.githubusercontent.com/18440102/128946381-abcba7ac-0ec5-447c-b70f-cfd314fc756c.png">

Basically we should show this banner when one or more chain is down (the down detection logic can be copied from https://www.figma.com/file/04OF6ZbTo0gLzmFm3kLRkS/app.emeris.com?node-id=4925%3A73468). We should adapt the text to something like:

If one chain is down:

"XXX chain appears to be down. Your balance on this chain may be unaccurate"

If two chains are down

"XXX and XXX chains appear to be down. Your balance on these chains may be unaccurate" 

If more than two chains are down

"Several chains appear to be down. Your balance on these chains may be unaccurate".  (+add list of chains on tooltip on hover)

## Option 2

Hide balances on down chains with warning
